src/ScaffoldInstallerPlugin.php

```diff
@@ -407,19 +407,30 @@ private function installTugboat(string $scaffoldPath): void {
 
         // Filter out unsupported config overrides.
         if (!empty($tugboatConfigOverride['php']) && is_array($tugboatConfigOverride['php'])) {
-            $tugboatConfigOverride['php'] = array_filter($tugboatConfigOverride['php'],
-                function($key) {
-                    return in_array($key,
-                        ['aliases', 'urls', 'visualdiff', 'screenshot']);
-                },
-                ARRAY_FILTER_USE_KEY);
+            $tugboatConfigOverride['php'] = array_filter($tugboatConfigOverride['php'], function($key) {
+                return in_array($key, ['aliases', 'urls', 'visualdiff', 'screenshot']);
+            }, ARRAY_FILTER_USE_KEY);
+
+            // Prepare YAML data
+            $yamlData = [];
+            foreach ($tugboatConfigOverride['php'] as $key => $value) {
+                $yamlData[$key] = $value;
+            }
```

Comment:
This seems wrong - as in, I suppose it works, but don't you just want something like:

```php
$yamlData = $tugboatConfigOverride['php'];
```